hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/HoodieSparkTable.java

```diff
@@ -63,9 +63,14 @@ protected HoodieSparkTable(HoodieWriteConfig config, HoodieEngineContext context
   public static <T extends HoodieRecordPayload> HoodieSparkTable<T> create(HoodieWriteConfig config, HoodieEngineContext context,
                                                                            boolean refreshTimeline) {
     HoodieTableMetaClient metaClient =
-        HoodieTableMetaClient.builder().setConf(context.getHadoopConf().get()).setBasePath(config.getBasePath())
-            .setLoadActiveTimelineOnLoad(true).setConsistencyGuardConfig(config.getConsistencyGuardConfig())
-            .setLayoutVersion(Option.of(new TimelineLayoutVersion(config.getTimelineLayoutVersion()))).build();
+        HoodieTableMetaClient.builder()
+                .setConf(context.getHadoopConf().get())
+                .setBasePath(config.getBasePath())
+                .setLoadActiveTimelineOnLoad(true)
+                .setConsistencyGuardConfig(config.getConsistencyGuardConfig())
+                .setFileSystemGuardConfig(config.getFileSystemGuardConfig())
```

Comment:
hi @zhangyue19921010 ，i think also require add about setFileSystemGuardConfig in HoodieFlinkTable calss

Reply:
Sure, thing. Will get it done.

Reply:
cc @danny0405 :)